stable/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 3.5.0
-appVersion: 0.15.1
+version: 4.0.0
+appVersion: 0.16.0
```

Comment:
Either upgrade Chart.yaml, values.yaml and README.md (and edit PR name) or revert to 0.15.1

Reply:
the application version is already at 0.16.0 in master branch, I guess it was omitted some time ago, see https://github.com/helm/charts/blob/master/stable/prometheus-blackbox-exporter/values.yaml#L16